app/helpers/application_helper.rb

```diff
@@ -1,2 +1,10 @@
 module ApplicationHelper
+  def full_title page_title
+    base_title = "Ruby on Rails Tutorial Sample App"
+    if page_title.empty?
```

Comment:
Dùng empty? có đúng không em nhỉ?
Phân biệt cho chị empty?, nil? và blank? nhé

Reply:
dạ em nghỉ trong trường hợp này thì dùng empty hoặc nil thì cũng đúng á chị .
Phân biệt 3 lệnh : 
1.empty? : Trả về true nếu  lenght của đối tượng = 0
2.nil? : Nó chỉ trả về true  nếu nó là nil  
3. blank? : Trả về true nếu đối tượng đó false , empty hoặc là nó chỉ toàn những khoảng trắng

Reply:
chưa đầy đủ.
ở đây dùng method blank? nhé.

Em nên tìm hiểu sâu hơn

Reply:
Dạ ok chị